keyboards/handwired/ms_sculpt_mobile/8x18_arm/8x18_arm.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include QMK_KEYBOARD_H
+//#include "ms_sculpt_mobile.h"
+//	#define LAYOUT(k00) {{ k00 }}
+
+#ifdef BOARD_GENERIC_STM32_F103
+#define LED_ON()    do { palClearPad(GPIOC, 13) ;} while (0)
```

Comment:
These should use <https://docs.qmk.fm/#/internals_gpio_control?id=functions> to which you should be able to ditch the `BOARD_GENERIC_STM32_F103` define.